lib/info.js

```diff
@@ -97,16 +97,18 @@ exports.getBasicInfo = (id, options, callback) => {
  * @return {Array.<Object>}
  */
 const parseFormats = (info) => {
-  let formats = [];
-  if (info.url_encoded_fmt_stream_map) {
-    formats = formats.concat(info.url_encoded_fmt_stream_map.split(','));
+  let formats = []
+
+  if (info.player_response.streamingData.formats) {
```

Comment:
is `streamingData` always defined? thinking that it should be checked if it's not